sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md

```diff
@@ -47,6 +47,34 @@ ArmOperation<CommunicationServiceResource> communicationServiceLro = await colle
 CommunicationServiceResource communicationService = communicationServiceLro.Value;
 ```
 
+***Create a Communication Service with User Assigned Managed Identity***
+
+```C# Snippet:Managing_CommunicationService_CreateAnApplication with User Assigned Managed Identity
+var communicationService = await communicationManagementClient.CommunicationServices.CreateAsync(
```

Comment:
Where did this communicationManagementClient come from?

Reply:
https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-net

Reply:
Yes, but I mean in the code. You just put it without ever declaring it